src/all/comicskingdom/src/eu/kanade/tachiyomi/extension/all/comicskingdom/ComicsKingdom.kt

```diff
@@ -0,0 +1,310 @@
+
+        val excluded: List<String>
+            get() = state.filter { it.isExcluded() }.map { it.gid }
+    }
+
+    override fun getFilterList() = FilterList(
+        OrderFilter(),
+        GenreList(getGenreList()),
+    )
+
+    override fun imageUrlParse(document: Document): String = ""
+
+    private fun getGenreList() = listOf(
+        Genre("Action", "action"),
+        Genre("Adventure", "adventure"),
+        Genre("Classic", "classic"),
+        Genre("Comedy", "comedy"),
+        Genre("Crime", "crime"),
+        Genre("Fantasy", "fantasy"),
+        Genre("Gag Cartoons", "gag-cartoons"),
+        Genre("Mystery", "mystery"),
+        Genre("New Arrivals", "new-arrivals"),
+        Genre("Non-Fiction", "non-fiction"),
+        Genre("OffBeat", "offbeat"),
+        Genre("Political Cartoons", "political-cartoons"),
+        Genre("Romance", "romance"),
+        Genre("Sci-Fi", "sci-fi"),
+        Genre("Slice Of Life", "slice-of-life"),
+        Genre("Superhero", "superhero"),
+        Genre("Vintage", "vintage"),
+    )
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        val compactpref = androidx.preference.CheckBoxPreference(screen.context).apply {
+            key = "compactPref"
+            title = "Compact chapters"
+            summary =
+                "Unchecking this will make each daily/weekly upload into a chapter which can be very slow because some comics have 8000+ uploads"
+            isChecked = true
+        }
+
+        screen.addPreference(compactpref)
+    }
+
+    private val preferences: SharedPreferences by lazy {
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
+    }
+
+    private fun shouldCompact() = preferences.getBoolean("compactPref", true)
+
+    override fun popularMangaSelector() = throw UnsupportedOperationException()
+    override fun latestUpdatesSelector() = throw UnsupportedOperationException()
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+
+    override fun popularMangaNextPageSelector() = throw UnsupportedOperationException()
+    override fun latestUpdatesNextPageSelector() = throw UnsupportedOperationException()
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+
+    override fun latestUpdatesFromElement(element: Element): SManga = throw UnsupportedOperationException()
+    override fun searchMangaFromElement(element: Element): SManga = throw UnsupportedOperationException()
+    override fun popularMangaFromElement(element: Element): SManga = throw UnsupportedOperationException()
+
+    override fun mangaDetailsParse(document: Document): SManga = throw UnsupportedOperationException()
+
+    override fun chapterListSelector() = throw UnsupportedOperationException()
+    override fun chapterFromElement(element: Element) = throw UnsupportedOperationException()
+    override fun pageListParse(document: Document) = throw UnsupportedOperationException()
```

Comment:
I didn't realize this previously... inherit from `HttpSource` instead of `ParsedHttpSource`